README.md

```diff
@@ -36,7 +37,7 @@ The `code` here is an ISO-639-2/B three-letter code. Here is the list of pre-ins
 - heb
 - ita
 - jpn
-- kor
+- kor (default analyzer: seunjeon_analyzer, alternative for k8s and on-premise deployment - nori)
```

Comment:
Is `seunjeon_analyzer` available for k8s and on-premise by default? Or they have to call API to change to `nori` after deployment?

Reply:
`seunjeon_analyzer` is available only for AWS Elasticsearch.
'nori' or `seunjeon_analyzer` are not included in the default analyzers set in elasticsearch,  they should be installed in the Dockerfile or in the elasticsearch using command `elasticsearch-plugin install ${plugin}`.